src/qibotn/backends/quimb.py

```diff
@@ -43,21 +48,23 @@ def configure_tn_simulation(
         self.max_bond_dimension = max_bond_dimension
         self.n_most_frequent_states = n_most_frequent_states
 
-    def setup_backend_specifics(self, qimb_backend="numpy"):
+    def setup_backend_specifics(self, qimb_backend="numpy", optimizer="auto-hq"):
         """Setup backend specifics.
         Args:
             qimb_backend: str
                 The backend to use for the quimb tensor network simulation.
+            optimizer: str, optional
+                The optimizer to use for the quimb tensor network simulation.
         """
         self.backend = qimb_backend
+        self.optimizer = optimizer
 
     def execute_circuit(
```

Comment:
As it is now, if we don't pass neither `nshots` nor `return_array` the function is doing nothing. Are we fine with this? It would be probably better to set as default one of the two options before.